hack/pman/rules.yaml

```diff
@@ -23,12 +23,6 @@ regex:
   - cluster-up/cluster/k8s-[0-9].[0-9][0-9]*
 regex_none:
   - cluster-up/cluster/kind*
-exclude:
-  - pattern: cluster-provision/centos8/*
-    exclude:
-      - 1.26-centos9
-      - 1.27
-      - 1.28
 specific:
```

Comment:
please also remove the `specific` section
instead `cluster-provision/centos9/*` should be added as an `all` rule

Reply:
OK will take a look

Reply:
Does it look ok now?

Reply:
yep thanks

Reply:
yep thanks
note that it is amended to the wrong commit